src/pages/expenses/clone/Clone.tsx

```diff
@@ -32,7 +33,10 @@ export function Clone() {
 
   const { documentTitle } = useTitle('new_expense');
   const { id } = useParams();
-  const { data } = useExpenseQuery({ id });
+  
+  const { data: dataExpense } = useExpenseQuery({ id });
+  const { data: dataReccuringExpense } = useRecurringExpenseQuery({ id });
+  const data = dataExpense || dataReccuringExpense;
```

Comment:
Why are we pouring the recurring expenses logic to expenses clone?

Apart from that, this is *old* way of cloning stuff. Please have a look at invoices and how's cloning done there. We extract the state into Jotai atom and perform cloning from there.

Reply:
Clone logic removed from expenses clone